x-pack/plugins/observability/public/pages/rule_details/components/actions.tsx

```diff
@@ -38,6 +38,11 @@ export function Actions({ ruleActions }: ActionsProps) {
     notifications: { toasts },
   } = useKibana().services;
   const { isLoadingActions, allActions, errorActions } = useFetchRuleActions({ http });
+  useEffect(() => {
+    if (errorActions) {
+      toasts.addDanger({ title: errorActions });
```

Comment:
@mgiota, Thank you for fixing that! Actually, I was thinking to move the toasts to here https://github.com/mgiota/kibana/blob/5155be43ffee5f5b18f6fe2eb1758898fd9d6776/x-pack/plugins/observability/public/hooks/use_fetch_rule_actions.ts#L36

Moving it inside the hook, means:
1. No need to add `useEffect` with the error check.
2. `useFetchRuleActions` will handle and show the error popup whenever it's used, not only on this page.
We could also change the `errorActions ` type to be `boolean` instead of `string` as we only need to know if there is an error.

What do you think?

Reply:
@mgiota, Ok, let's not block this PR and keep the implementation as it is.
It's important to have a consistent way of handling/showing errors across O11y pages, let's discuss for that during team time.